### ISSUE TITLE
Tests for twisted.runner should conform to the new style of naming test classes

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7726]
|[<img alt="eeshangarg's avatar" src="https://avatars.githubusercontent.com/u/7251823?s=50" width="50" height="50">](https://github.com/eeshangarg)| @eeshangarg reported|
|-|-|
|Trac ID|trac#7726|
|Type|enhancement|
|Created|2014-11-28 11:09:00Z|
|Branch|https://github.com/twisted/twisted/tree/runner-testcase-names-7726|

This ticket is a follow-up to ticket [ticket:7032].

Attachments:

* [bugfix_7726.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/d86/d86503ab2ec9d4e6b2503a56898030563e1aa8e3/7e09b25999dc1f0b099d748b8fb6f6e460c56f02.patch) (607 bytes) - added by eeshangarg on 2014-11-28 11:12:25Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__7726 7726
type__enhancement enhancement
reporter__eeshangarg eeshangarg
priority__normal normal
milestone__ 
branch__branches_runner_testcase_names_7726 branches/runner-testcase-names-7726
branch_author__adiroiban adiroiban
status__closed closed
resolution__fixed fixed
component__runner runner
keywords__easy easy
time__1417172940762181 1417172940762181
changetime__1422302428495201 1422302428495201
version__None None
owner__adiroiban adiroiban
cc__eeshangarg
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="eeshangarg's avatar" src="https://avatars.githubusercontent.com/u/7251823?s=50" width="50" height="50">](https://github.com/eeshangarg)<a name="note_1"></a>|@eeshangarg removed owner|
|-|-|

Hi!

I just added a patch(bugfix_7726.patch​) that changes all of the tests in twisted.runner to use the new style of naming test classes as discussed in ticket [ticket:7032].

Thanks & regards,
Eeshan

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_2"></a>|@adiroiban commented|
|-|-|

(In [43751]) Branching to runner-testcase-names-7726.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban set owner to @adiroiban|
|-|-|

Thanks for working on this.
I have created a branch and waiting for buildbot results.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_5"></a>|@adiroiban set status to `closed`|
|-|-|

(In [43770]) Merge runner-testcase-names-7726: Rename test cases from twisted.runner.

Author: eeshangarg
Reviewers: adiroiban
Fixes: [#7726](https://github.com/twisted/twisted/issues/7726)
